src/modules/timeline.js

```diff
@@ -163,36 +163,40 @@ export default function (playerInstance, options) {
     };
 
     playerInstance.setupThumbnailPreview = () => {
-        if (playerInstance.displayOptions.layoutControls.timelinePreview &&
-            (typeof playerInstance.displayOptions.layoutControls.timelinePreview.file === 'string') &&
-            (typeof playerInstance.displayOptions.layoutControls.timelinePreview.type === 'string')) {
-
-            if (playerInstance.displayOptions.layoutControls.timelinePreview.type === 'VTT') {
-                let eventOn = 'mousemove';
-                let eventOff = 'mouseleave';
-                if (playerInstance.mobileInfo.userOs) {
-                    eventOn = 'touchmove';
-                    eventOff = 'touchend';
-                }
-                document.getElementById(playerInstance.videoPlayerId + '_fluid_controls_progress_container')
-                    .addEventListener(eventOn, playerInstance.drawTimelinePreview.bind(playerInstance), false);
-                document.getElementById(playerInstance.videoPlayerId + '_fluid_controls_progress_container')
-                    .addEventListener(eventOff, function (event) {
-                        const progress = document.getElementById(playerInstance.videoPlayerId + '_fluid_controls_progress_container');
-                        if (typeof event.clientX !== 'undefined' && progress.contains(document.elementFromPoint(event.clientX, event.clientY))) {
-                            //False positive (Chrome bug when fast click causes leave event)
-                            return;
-                        }
-                        document.getElementById(playerInstance.videoPlayerId + '_fluid_timeline_preview_container').style.display = 'none';
-                        document.getElementById(playerInstance.videoPlayerId + '_fluid_timeline_preview_container_shadow').style.display = 'none';
-                    }, false);
-                playerInstance.generateTimelinePreviewTags();
+        const timelinePreview = playerInstance.displayOptions.layoutControls.timelinePreview;
```

Comment:
This can be simplified:

```javascript
const hasPreviewConfiguration = !!playerInstance.displayOptions.layoutControls.timelinePreview;
const hasPreviewType = !!playerInstance.displayOptions.layoutControls.timelinePreview.type;
if (!hasPreviewConfiguration || !hasPreviewType) {
  return;
}
```